### PR TITLE
Styled the landing page

### PR DESCRIPTION
**Pull request by lisa-sa-li**
Still unsure about the static background-image attribute in the css. Worse case I add a style tag in the html.

### CODE REVIEW

File: hackathon_site/event/static/event/styles/scss/styles.scss [hunk 1]
```diff
@@ -28,33 +40,102 @@ strong {
...
+    background-position: right top;
+    padding: 30px 0 50px;
+    margin-top: -64px;
+
+    @media only screen and (min-width: 2600px) {
+        background-position: left top;
+    }
+
+    @include responsive(sm-down) {
+        background-image: url("/static/event/images/purple_landing_mobile.svg");
```
Comment (grahamhoyes) [inline hackathon_site/event/static/event/styles/scss/styles.scss:114]: Ditto about this breaking

File: hackathon_site/event/jinja2/event/landing.html [hunk 2]
```diff
@@ -27,292 +29,296 @@
...
                 </h5>
             </div>
+            {% if localtime(now) > localtime(registration_open_date) %}
             <div class="row">
-                <a class="btn-large waves-effect waves-light colorBtn">Login/Apply</a>
+                <a class="btn-large waves-effect waves-light secondaryColor coloredBtn">
+                    {% if localtime(now) > localtime(registration_close_date) %}
+                    Login
+                    {% else %}
+                    Login/Apply
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:66]: Just a note about this for the future: We'll have to do some logic when they log in, and either send them to the application page or dashboard depending whether they've filled out an application. Should be easy.

File: hackathon_site/event/jinja2/event/landing.html [hunk 3]
```diff
@@ -27,292 +29,296 @@
...
-                <p class="center">
-                    Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto
-                    beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi
-                    nesciunt.
-                </p>
-            </div>
+                    <p class="center white-text">
+                        Join us at NewHacks to transform your ideas into reality! In this 24-hour event, you and your team will make a project from scratch using software and present your project to a panel of industry
+                        judges to compete for prizes. This year, the theme for NewHacks is
+                        <span class="lightText">digital education</span> and <span class="lightText">life convenience</span> - so be creative with software solutions to improve online learning or quality of life!
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:84]: image

This emphasis color doesn't stand out very much, at least not with the positioning of things on my screen.

File: hackathon_site/event/jinja2/event/landing.html [hunk 4]
```diff
@@ -27,292 +29,296 @@
...
+                    <div class="scheduleRow">
+                        <p class="lightText scheduleTime">4PM</p>
+                        <p class="white-text">End</p>
+                    </div>
                 </div>
             </div>
-        </div>
-        <div class="m12 center">
-            <a href="#" class="btn-large waves-effect waves-light colorBtn" rel="noopener" target="_blank">Full schedule</a>
+            <div class="m12 center">
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:168]: Given that this is going live soon, may want to hide / remove this button until it actually does something

File: hackathon_site/event/jinja2/event/landing.html [hunk 5]
```diff
@@ -27,292 +29,296 @@
...
+                    <h3 class="center white-text">FAQ</h3>
+                </div>
+                <div class="col s12 m6">
+                    <ul class="collapsible">
+                        <li>
+                            <div class="collapsible-header">
+                                What is a hackathon?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>A hackathon is a big bonanza where people from everyone doing everything come together to create projects together with the newest and most exciting tech.</span>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:187]: "where people from everyone doing everything"

File: hackathon_site/event/jinja2/event/landing.html [hunk 6]
```diff
@@ -27,292 +29,296 @@
...
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                How much does it cost?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>$0! IEEE UofT is a non-profit organization that aims to provide students accessible technical events.</span>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:195]: Are we actually a non-profit? Can we give charitable recepits for sponsorships? Just wondering about the legal language

File: hackathon_site/event/jinja2/event/landing.html [hunk 7]
```diff
@@ -27,292 +29,296 @@
...
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                When can I expect to hear back after I applied?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    We are having 2 rounds of acceptance. If you applied early, you might get an email on the weekend of Oct 10th/11th. If you still haven’t gotten any replies by then, don’t worry! We
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:204]: "We are having 2 rounds of acceptance." -> "There will be 2 rounds of acceptances"

"second round of acceptance" -> "second round of acceptances"

File: hackathon_site/event/jinja2/event/landing.html [hunk 8]
```diff
@@ -27,292 +29,296 @@
...
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                What equipment and support would be provided?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Unfortunately, due to the current situation of COVID19. We won’t be able to provide any hardware. However, there would be lots of mentors ready to help you! During the online
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:231]: Should be a comma after COVID19, not a fullstop.

The use of the word "would" implies that that would have been the case, but won't. Should it be replaced with "will"?

File: hackathon_site/event/jinja2/event/landing.html [hunk 9]
```diff
@@ -27,292 +29,296 @@
...
-
 <div class="carousel">
-    <!-- IMAGES MUST ALL BE THE SAME SIZE -->
-    <a class="carousel-item" ><img src="https://i.imgur.com/UoVt0SD.jpg" alt="2019-image-1"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/tFJUdpY.jpg" alt="2019-image-2"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/EeFW9yt.jpg" alt="2019-image-3"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/ho1TpX3.jpg" alt="2019-image-4"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/NuVyjXK.jpg" alt="2019-image-5"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/r7ZMttJ.jpg" alt="2019-image-6"/></a>
+    <a class="carousel-item" href="#one!"><img src="https://i.imgur.com/leuHG62.jpg" alt="2019-image-1" /></a>
```
Comment (grahamhoyes) [inline hackathon_site/event/jinja2/event/landing.html:295]: Are you sure these hrefs needed to be added back? Seems to work without them, and it's strange to have them in the browser hash if you click an image

File: hackathon_site/event/jinja2/event/landing.html (hunk 4 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/landing.html:168]: I asked logistics for a docs to link and they'll send it soon

Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/landing.html:168]: eh i'll put it back once they get back to me

File: hackathon_site/event/jinja2/event/landing.html (hunk 5 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/landing.html:187]: yeah wtf was Raymond trying to say

File: hackathon_site/event/jinja2/event/landing.html (hunk 9 above)
Comment (lisa-sa-li) [reply hackathon_site/event/jinja2/event/landing.html:295]: They dont work for me when I remove the href

**Pull request merged**